static/app/components/u2f/u2finterface.tsx

```diff
@@ -5,12 +5,13 @@ import * as cbor from 'cbor-web';
 import {base64urlToBuffer, bufferToBase64url} from 'sentry/components/u2f/webAuthnHelper';
 import {t, tct} from 'sentry/locale';
 import ConfigStore from 'sentry/stores/configStore';
-import type {ChallengeData} from 'sentry/types';
+import type {ChallengeData, StateData} from 'sentry/types';
 
 type TapParams = {
   challenge: string;
   response: string;
   isSuperuserModal?: boolean;
+  state?: string;
```

Comment:
state is very generic.... also kind of a confusing attr to pass, especially for a react component
can we call this soemthing else?

maybe tapState at least?

Reply:
Ah wait i see...

we're typing the arguments for the onTap helper 🤔 

would still prefer if we did not call this `state` for the onTap unless we're actually passing the full state object in. which it looks like we're not

Reply:
I can call it `authState` and rename it everywhere